### DISCUSSION TITLE
Orion Trapezium - some weirdness

### DISCUSSION THREAD

**Discussion by firsttest12** [Stellarium/stellarium#3152] [General]
Clicking on the four bright spots that represent the stars / binary-starts that comprise the Trapezium in Orion does not reliably bring up their names. Specifically:

- if you click in the middle of Theta-Orion-A you'll get it, but just slightly below centre the C will be highlighted instead;
- clicking on B does nothing (nor does it appear using the search function, whilst A, C, D do);
- the way to highlight C seems to NOT be clicking centre, but below and outside the bright area;
- clicking on A just works;

Not sure if this weirdness is related to the presence of binary stars. But I can't figure out if there's an error, where it might be?

**Comment by gzotti** [comment]
#3136
